test/src/org/exist/xupdate/modifications/append_child.xml

```diff
@@ -5,7 +5,7 @@
 		<xu:element name="email"><xu:attribute name="type">home</xu:attribute>andrew@foo.org</xu:element>
 	</xu:append>
 	
-	<xu:append select="//address[lname = 'Jekyll']" child="3">
+	<xu:append select="//address[lname = 'Jekyll']" child="5">
```

Comment:
ok, this is around WS change too, adding 2 text nodes between address - fname & fname - lname
```
	<address>
		<fname>John</fname>
		<lname>Jekyll</lname>
		<email type="home">jekyll@foo.org</email>
		<company>Johnny's Car Wash</company>
		<city>Bloomington</city>
		<email type="work">john@carwash.com</email>
	</address>
```